.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Release to npm
+
+on:
+  release:
+    types: [created]
```

Comment:
What is this trigger to do a release?

I usually use
```
on:
  push:
    tags:
      - '*'
```

So a release is triggered when I create a tag.

Reply:
@cah-briangantzler I removed `publish.yml` from the PR and we can get back to publish automation later